syscalls.go

```diff
@@ -87,9 +87,11 @@ var haveMapMutabilityModifiers = internal.NewFeatureTest("read- and write-only m
 		MaxEntries: 1,
 		MapFlags:   unix.BPF_F_RDONLY_PROG,
 	})
+
```

Comment:
Nit: gratuitous whitespace change. It's fine to change whitespace to your preference when you're editing a function anyways, but this just increases the diff.